public/404.html

```diff
@@ -1,67 +1,135 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
+
 <head>
-  <title>The page you were looking for doesn't exist (404)</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  <style>
-  .rails-default-error-page {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
-
-  .rails-default-error-page div.dialog {
-    width: 95%;
-    max-width: 33em;
-    margin: 4em auto 0;
-  }
-
-  .rails-default-error-page div.dialog > div {
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #BBB;
-    border-top: #B00100 solid 4px;
-    border-top-left-radius: 9px;
-    border-top-right-radius: 9px;
-    background-color: white;
-    padding: 7px 12% 0;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
-
-  .rails-default-error-page h1 {
-    font-size: 100%;
-    color: #730E15;
-    line-height: 1.5em;
-  }
-
-  .rails-default-error-page div.dialog > p {
-    margin: 0 0 1em;
-    padding: 1em;
-    background-color: #F7F7F7;
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #999;
-    border-bottom-left-radius: 4px;
-    border-bottom-right-radius: 4px;
-    border-top-color: #DADADA;
-    color: #666;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
-  </style>
+  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+  <meta charset="utf-8">
+  <title>Page Not Found - Response to Claim - MoJ</title>
+
+  <!--[if gt IE 8]><!-->
+  <link rel="stylesheet" media="screen" href="404_files/govuk-template.css" crossorigin="anonymous">
+  <!--<![endif]-->
+  <link rel="stylesheet" media="print" href="404_files/govuk-template-print.css" crossorigin="anonymous">
+
+  <!--[if gte IE 9]><!-->
+  <link rel="stylesheet" media="all" href="404_files/fonts.css" crossorigin="anonymous">
+  <!--<![endif]-->
+
+  <meta name="theme-color" content="#0b0c0c">
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link rel="stylesheet" media="all" href="404_files/confirmation_supplied_details.css" data-turbolinks-track="reload">
+  <link rel="stylesheet" media="all" href="404_files/elements.css" data-turbolinks-track="reload">
+  <link rel="stylesheet" media="all" href="404_files/application.css" data-turbolinks-track="reload">
+  <script src="404_files/jquery.js"></script>
```

Comment:
Do we need many of these ?  we dont need turbolinks, show-hide-content, rails-ujs, cation-cable or even application i doubt ?

Reply:
Removed in 81a7f01